packages/yourloops/test/unit/components/alarm/alarms-content-configuration.test.tsx

```diff
@@ -297,4 +273,138 @@ describe('AlarmsContentConfiguration', () => {
     expect(screen.queryByText('default-min-max')).not.toBeNull()
     expect(screen.queryAllByText('default')).toHaveLength(4)
   })
+  it('save button should be disabled when low bg value is not integer', () => {
+    const incorrectMonitoring = { ...monitoring }
+    incorrectMonitoring.parameters.lowBg = 60.5
+    renderTeamAlarmsContent({ monitoring: incorrectMonitoring, onSave, saveInProgress: false })
+    expect((document.getElementById('low-bg-text-field-id-helper-text') as HTMLInputElement)).toHaveTextContent('mandatory-integer')
+    checkSaveButtonDisabled()
+  })
+
+  it('save button should be disabled when high bg value is not integer', () => {
+    const incorrectMonitoring = { ...monitoring }
+    incorrectMonitoring.parameters.highBg = 145.5
+    renderTeamAlarmsContent({ monitoring: incorrectMonitoring, onSave, saveInProgress: false })
+    expect((document.getElementById('high-bg-text-field-id-helper-text') as HTMLInputElement)).toHaveTextContent('mandatory-integer')
+    checkSaveButtonDisabled()
+  })
+  it('save button should be disabled when very low bg value is not integer', () => {
+    const incorrectMonitoring = { ...monitoring }
+    incorrectMonitoring.parameters.veryLowBg = 40.5
+    renderTeamAlarmsContent({ monitoring: incorrectMonitoring, onSave, saveInProgress: false })
+    expect((document.getElementById('very-low-bg-text-field-id-helper-text') as HTMLInputElement)).toHaveTextContent('mandatory-integer')
+    checkSaveButtonDisabled()
+  })
+  // if is function return false
+  it('should return false if the value is within the low target and is in mg/dL', () => {
```

Comment:
All the following tests should be in a dedicated test file named `alarms-content-configuration.hook.test` or `alarms-content-configuration.util.test` (you should have both and put each test in the correct one :slightly_smiling_face: )